src/vm/ilmarshalers.h

```diff
@@ -2832,26 +2939,42 @@ class ILAsAnyMarshalerBase : public ILMarshaler
         c_CLRSize               = sizeof(OBJECTREF),
     };
 
-    ILAsAnyMarshalerBase() :
-        m_dwMarshalerLocalNum(-1)
-    {
-        LIMITED_METHOD_CONTRACT;
-    }
-
 protected:
     static const BYTE ML_IN  = 0x10;
```

Comment:
Can you move both `ML_IN` and `ML_OUT` to usage and document why the values are what they are.